django/contrib/sitemaps/management/commands/ping_google.py

```diff
@@ -1,16 +1,19 @@
-from django.contrib.sitemaps import ping_google
+import warnings
+
 from django.core.management.base import BaseCommand
+from django.utils.deprecation import RemovedInDjango60Warning
 
 
 class Command(BaseCommand):
-    help = "Ping Google with an updated sitemap, pass optional url of sitemap"
+    help = "Ping Google with an updated sitemap (non-operational and deprecated)"
```

Comment:
Not sure if it is best to just note the deprecation in the help text, or to print something to stdout also?

Reply:
I think that's done with the call to `warnings.warn()` in l. 15